src/core/nixl_agent.cpp

```
@@ -613,13 +613,18 @@ nixlAgent::prepXferDlist (const std::string &agent_name,
     for (const auto &backend : *backend_set) {
-        nixl_meta_dlist_t dlist(descs.getType());
+        int stride_count;
+        nixl_stride_dlist_t dlist(descs.getType());
         if (section.populate(descs, backend, dlist) == NIXL_SUCCESS) {
-            dlists.try_emplace(backend, std::make_unique<nixl_meta_dlist_t>(std::move(dlist)));
+            stride_count = dlist.descCount();
+            dlists.try_emplace(backend, std::make_unique<nixl_stride_dlist_t>(std::move(dlist)));
         }
+
+        NIXL_DEBUG << "backend " << backend->getType() << ": prepared " << descs.descCount()
+                   << " descriptors into " << stride_count << " strides";
```

Cppcheck (2.20.0)

[warning] Uninitialized variable

(uninitvar)